layouts/ddsql_schema/section.html

```diff
@@ -0,0 +1,332 @@
+{{ define "main" }}
```

Comment:
Hey! Sorry, I just saw that the PR was merged, and I realized too late that we are showing "experimental" datasets to customers. Would it be possible to add a filter on the datasets shown where they need to have both `status: stable` and `variant: public` in order to be shown?

Also, we will need to update the metadata of the datasets on our side, so you will need to "re-ingest" the YAML files. Could you let us know how the YAML files are updated WRT the docs? Is this a manual or automatic process? Thank you!